data/XML/groups.xml

```diff
@@ -16,7 +141,54 @@
 			<flag hasinfinitemana="1" />
 			<flag hasinfinitesoul="1" />
 			<flag hasnoexhaustion="1" />
+			<flag cannotusespells="1" />
+			<flag cannotpickupitem="1" />
 			<flag canalwayslogin="1" />
+			<flag canbroadcast="1" />
+			<flag canedithouses="1" />
+			<flag cannotbebanned="1" />
```

Comment:
`cannotbebanned` should be 0

Reply:
There was once a gamemaster called Deathwalker that was banished while still a gamemaster. Just for the curious :stuck_out_tongue: